source/agora/common/Config.d

```diff
@@ -174,6 +174,13 @@ public struct NodeConfig
 
     /// The factor to calculate for the fee of data payload
     public uint tx_payload_fee_factor = 200;
+
+    /// The share that Validators would get out of the transction fees
+    /// Out of 100
+    public ubyte validator_tx_fee_cut = 70;
+
+    /// How frequent the payments to Validators will be
```

Comment:
should mention this is the number of blocks (?). Is 144 from the whitepaper?

Reply:
144 was @Geod24's suggestion.